packages/@stimulus/core/src/application.ts

```diff
@@ -71,6 +76,24 @@ export class Application implements ErrorHandler {
   handleError(error: Error, message: string, detail: object) {
     this.logger.error(`%s\n\n%o\n\n%o`, message, error, detail)
   }
+
+  // Debug logging
+
+  logDebugActivity = (identifier: string, functionName: string, detail: object = {}): void => {
+    if (this.debug) {
+      this.logFormattedMessage(identifier, functionName, detail)
+    }
+  }
+
+  private logFormattedMessage(identifier: string, functionName: string, detail: object = {}) {
+    const darkMode = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
can we use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here to shorten the condition?

Reply:
I can try and see if it passes the CI